client/src/i18n/locales/en/index.json

```diff
@@ -185,5 +185,17 @@
   "sept": "Sept",
   "oct": "Oct",
   "nov": "Nov",
-  "dec": "Dec"
+  "dec": "Dec",
+
+  "definitions": "Definitions",
+  "exceededLimitDef": "The child has attended more days than they were approved for this month. You have earned the max monthly payment and will not be paid for additional days.",
+  "onTrackDef": "The child is attending most or all of their days and you will likely receive most of the subsidy payment this month.",
+  "atRiskDef": "The child has been absent frequently and you may not receive the full subsidy payment this month.",
+  "fullDaysDef": "Days with 6-10 hours attended. Approved days come from the approval letter.",
+  "hoursDef": "Days with less than 6 hours attended.",
+  "absencesDef": "You can be paid for up to five absences. Absences are included in your revenue calculations.",
+  "earnedRevenueDef2": "The revenue you have already earned this month, based on the child’s attendance. This does not include other subsidy revenue such as transportation.",
```

Comment:
Can we give this block of definitions (and probably the ones from IL) more meaningful names? `neEarnedRevenueDef` and `ilEarnedRevenueDef` or something similar to that?  Would it make sense (at some point, not necessarily in the scope of this ticket) to refactor these locale files to have a more nested structure, i.e. group universal text blocks under "universal", then nest all the nebraska definitions under a "nebraskaDefinitions" key, etc.?

Reply:
yes! I like the idea of the refactor. perhaps, we can create a ticket for that. in the meantime, I will remain these to be 'ne' specific!